architecture-decision-records.md

```diff
@@ -0,0 +1,15 @@
+# ADRs
+
+Software engineering and system design are often a collaboration effort and, in particular, the decisions that shape a project often arise as [RFCs](RFCs.md), but once the RFC has been... well, commented on, how do we articulate and present the decisions that come of out them? 
+
+Enter the architecture decision records, ADRs for short.
+
+Joel Parker Henderson has a [detailed presentation of ADRs](https://github.com/joelparkerhenderson/architecture-decision-record), but for the purpose of this text, an architecture decision record (ADR) is a document that captures an important architectural decision made along with its context and consequences.
```

Comment:
GitHub have a pretty good document on this too: https://adr.github.io/

Reply:
Excellent. Added that one too :)